### PR TITLE
[ET-2077] Free Tickets: Show "Free" instead of "$0.00"

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket
[ET-2077]

### 🗒️ Description
We want to show the ticket price as "Free" instead of "$0.00"

### 🎥 Artifacts 
image
image

### ✔️ Checklist
- [x] Changelog entry in the `readme.txt` file. (added in previous PR)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [x] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[ET-2077]: https://stellarwp.atlassian.net/browse/ET-2077?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Camwyn** [comment]
I am against putting logic in the template. 

`get_currency()` has hooks - aren't those useable here?

File: src/views/v2/commerce/ticket/regular-price.php [hunk 1]
```diff
@@ -23,4 +23,11 @@
 	return;
 }
 
-echo esc_html( $price->get_currency() );
+$regular_price_label = $price->get_currency();
+
+// If the price is zero, we should display it as free.
+if ( $price->get_decimal() == 0 ) {
+	$regular_price_label = esc_html__( 'Free', 'event-tickets' );
```
Comment (Camwyn) [inline src/views/v2/commerce/ticket/regular-price.php:30]:
```suggestion
	$regular_price_label = esc_html_x( 'Free', "Text displayed for a free ticket.", 'event-tickets' );
```
Or etc...add context.

**Review by Camwyn — CHANGES_REQUESTED** [review]

File: src/Tickets/Commerce/Utils/Value.php [hunk 2]
```diff
@@ -75,4 +75,19 @@ public function get_shortcode_price_html() {
...
+	/**
+	 * Get the display currency.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The display text for this value.
+	 */
+	public function get_currency_display() {
+		if ( $this->get_decimal() == 0 ) {
+			return _x( 'Free', 'No cost', 'event-tickets' );
```
Comment (Camwyn) [inline src/Tickets/Commerce/Utils/Value.php:88]: I'd add a filter. Post-translation. 
`get_currency()` [already has one](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tribe/Tickets.php#L4253) so I'm not as concerned about that. But we should let folks change this text if they want.

Comment (codingmusician) [reply src/Tickets/Commerce/Utils/Value.php]: Done. Thanks!

**Review by Camwyn — APPROVED** [review]

**Pull request merged**